src/rtcModule/webrtc.cpp

```diff
@@ -87,15 +89,31 @@ RtcModule::RtcModule(karere::Client& client, IGlobalHandler& handler,
 
 void RtcModule::init()
 {
-    StaticProvider iceServerStatic(mStaticIceSever);
+    string ipv4;
+    string ipv6;
+    StaticProvider iceServerStatic;
+    mKarereClient.mDnsCache.getIp(TURNSERVER_SHARE, ipv4, ipv6);
+    if (ipv4.size() || ipv6.size())
+    {
+        iceServerStatic = getTurnServerProvider(ipv4, ipv6);
```

Comment:
If possible, it would be better to reuse the code to `setServers()`, perhaps by adding the IPs as optional param. It could be not a good idea to prepare the JSON with fixed values. The API returns the service (`turn`), the port (`3478`) and even the protocol (`udp`). Ideally, we should keep using the values returned by the API originally, so in case the API needs to adjust any parameter, we don't need changes.
Maybe the returned JSON could be stored in the URL column in the `dns_cache` DB table. So it can be used here, and if the IPs are also in cache, use then instead of the domain name. What do you think, @alber2510 ??

Reply:
I tried in that way but our `Url` is not prepared to manage `turn:trn530n002.karere.mega.nz:3478?transport=udp` as a url. `turn:` is not a valid part for a url. But I can removed that part and keep the rest 
I'll modify the PR

Reply:
Fixed